src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -17,49 +17,42 @@ function getErrorMessage(field, message) {
  * `reviewSection` - Boolean. Hides components that are only needed for ReviewAndSubmitSection.
  */
 class AnnualIncomeSection extends React.Component {
-  componentWillMount() {
-    this.props.initializeChildIncomeFields();
-  }
-
   // TODO: Figure out best way to enable users to change their response to pension
   render() {
     const message = 'Please enter only numbers and a decimal point if necessary (no commas or currency signs)';
     let childrenIncomeInput;
     let childrenIncomeReview;
     let content;
 
-    if (this.props.data.hasChildrenToReport === true) {
+    if (this.props.data.hasChildrenToReport.value === 'Y') {
       childrenIncomeInput = (
         <div className="input-section">
           <FixedTable
               component={ChildIncome}
-              onRowsUpdate={(update) => {this.props.onStateChange('childrenIncome', update);}}
-              relatedData={this.props.data.children}
-              rows={this.props.data.childrenIncome}/>
+              onRowsUpdate={(update) => {this.props.onStateChange('children', update);}}
+              rows={this.props.data.children}/>
         </div>
       );
     }
 
     if (this.props.isSectionComplete && this.props.reviewSection) {
-      const childrenData = this.props.data.children;
-      let reactKey = 0;
-      childrenIncomeReview = this.props.data.childrenIncome.map((obj, index) => {
+      childrenIncomeReview = this.props.data.children.map((child, index) => {
         return (
-          <div key={reactKey++}>
-            <h6>Child: {`${childrenData[index].childFullName.first.value} ${childrenData[index].childFullName.last.value}`}</h6>
+          <div key={`child-${index}`}>
+            <h6>Child: {`${child.childFullName.first.value} ${child.childFullName.last.value}`}</h6>
             <table className="review usa-table-borderless">
               <tbody>
                 <tr>
                   <td>Children Gross Income:</td>
-                  <td>{obj.childGrossIncome.value}</td>
+                  <td>{child.income.grossIncome.value}</td>
```

Comment:
I think these are incorrect, looks like it's just `child.grossIncome` based on the store?